src/ring_theory/ideal/quotient.lean

```diff
@@ -410,4 +411,25 @@ noncomputable def quotient_inf_ring_equiv_pi_quotient [finite ι] (f : ι → id
 
 end chinese_remainder
 
+/-- The product over `fin 2` of some rings is just the cartesian product of these rings. -/
+@[simps]
+def _root_.ring_equiv.fin_two (R : fin 2 → Type*) [Π i, semiring (R i)] :
+  (Π (i : fin 2), R i) ≃+* R 0 × R 1 :=
+{ to_fun := pi_fin_two_equiv R,
+  map_add' := λ a b, rfl,
+  map_mul' := λ a b, rfl,
+  .. pi_fin_two_equiv R }
+
+/-- **Chinese remainder theorem**, specialized to two ideals. -/
+noncomputable def quotient_inf_equiv_quotient_prod (I J : ideal R)
```

Comment:
I think there should be (simp?) lemmas showing that the compositions of this and the projections are `ideal.quotient.factor`, and the elementwise versions of them.